frontend/src/App.tsx

```diff
@@ -18,7 +22,20 @@ function App() {
           element={<CurrentSongView asyncUser={asyncUser} />}
         />
         <Route path="/live/:sessionKey/add-song" element={<AddSongPage />} />
-        <Route path="/live" element={<WelcomePage asyncUser={asyncUser} />} />
+        <Route
+          path="/live/"
+          element={<WelcomePage asyncUser={asyncUser} songbooks={songbooks} />}
+        >
+          <Route path="/live/" element={<WishlistForm />} />
```

Comment:
I think this makes the path for wishlist `/live/live`... though maybe not since you're using an absolute path. For readability I would remove the path and instead just use `index` (no value needed, it'll default to true)